### ISSUE TITLE
Missing some functions on our Interpreter(e.g. IfStmt, Gt, IfElseStmt...)

### ISSUE THREAD

**Issue by luisamaralh** [UnB-CIC/sle-course#29]

**Comment by rbonifacio** [comment]
Yes.... we definitely should work on this issue. Perhaps @luisamaralh and @waltim could work on this. I am still missing a couple of fundamental expressions (e.g., and, or, not, >, <, ==, >=, <=, multiplication, and division).

**Comment by mateusluizfb** [comment]
I think the `invokeReturn` is missing too, or you guys implemented in another module?
